apis/aws.go

```diff
@@ -22,6 +22,8 @@ import (
 
 	ec2v1alpha1 "github.com/crossplane/provider-aws/apis/ec2/v1alpha1"
 	ec2v1beta1 "github.com/crossplane/provider-aws/apis/ec2/v1beta1"
+	eksv1alpha1 "github.com/crossplane/provider-aws/apis/eks/v1alpha1"
```

Comment:
why do we need both `eksv1alpha1` and `eksv1beta1`? Is `eksv1beta1` enough?

Reply:
NodeGroup is in eksv1alpha1.